README.MD

```diff
@@ -15,7 +15,7 @@ While this program will copy your resource packs before converting them, we stil
 ## Usage
 [Download the compiled jar file](https://github.com/agentdid127/ResourcePackConverter/releases/latest), or compile the source yourself.  
 The program will look for any valid resource packs in the current directory and is easily run by doing this.
-(Converts from 1.12.x to 1.13.x)
+(Converts from 1.8.x to 1.20.x)
```

Comment:
Should be 1.12.x to 1.13.x, as it's the default conversion according to the frontend implementations.

Reply:
Fixed already